README.md

```diff
@@ -67,7 +67,33 @@ The output directory will include a python module that corresponds to the
 original module. This code depends on the following python modules:
 
 - [jsii](https://pypi.org/project/jsii/)
-- [publication](https://pypi.org/project/publication/)
+
+### Java Output
+
+To produce a Java module from your source, use the `java` option:
+
+```ts
+await srcmak('srcdir', {
+  java: {
+    outdir: '/path/to/project/root',
+    package: 'hello.world'
+  }
+});
+```
+
+Or the `--java-*` switches in the CLI:
+
+```bash
+$ jsii-srcmak /src/dir --java-outdir=dir --java-package=hello.world
+```
+
+* The `outdir`/`--java-outdir` option points to the root directory of your Java project.
+* The `package`/`--java-package` option is the java package name.
+
+The output directory will include a java module that corresponds to the
+original module. This code depends on the following maven package (should be defined directly or indirectly in the project's `pom.xml` file):
+
+- [jsii](https://mvnrepository.com/artifact/software.amazon.jsii)
```

Comment:
Also worth mentioning that the output will include a tarball resource that must be bundled in the project (maybe provide an example snippet from a `pom.xml`?)